user/travis-pro.md

```diff
@@ -61,3 +61,16 @@ Travis CI is, and always will be, free for open source projects.
 
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
+
+## Making a private repository public
```

Comment:
Since the other topics are in the form of questions, I think this should be a question too. WDYT?